### PR TITLE
add project review repo

### PR DESCRIPTION
**Pull request by AmineDerbal**
In this milestone : 

- I will review my code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,141 @@
...
+          id="txtarea"
+          class="text-area"
+          name="comments"
+          placeholder="enter your comment here"
+        ></textarea>
+      </div>
+    </form>
+    <div class="form-group">
+      <button id="submit" type="submit">Submit</button>
+    </div>
```
Comment (AmineDerbal) [inline index.html:142]: - [ ] Kindly use semantic elements for the form, put <form id="survey-form"> in a section tag

**Review by AmineDerbal — COMMENTED** [review]
**Missing**

-  No linters.

- No GithubFlow.
- no Readme file

**Suggestions**

- Kindly set up the linters and github flows, check this [https://github.com/microverseinc/linters-config](page) to do it
- Kindly add a readme file , use this [https://github.com/microverseinc/readme-template](template)

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @AmineDerbal, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- polite comment ✔️
- inline comment ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] I would advise you to give reasons for your required changes. I can see you commented on semantic elements for the form. This is for the student to understand your point of view as the reviewer and not be acting blindly.
- [ ] I can see you made a list of changes, it is necessary for you to explain so that the student can have an adequate understanding of what you mean and not work in the dark.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmineDerbal — COMMENTED** [review]
**Missing**

-  No linters.

- No GithubFlow.
- no Readme file

**Suggestions**

- Kindly set up the linters, linter is a tool that programmatically scans your code with the goal of finding issues that can lead to bugs or inconsistencies with code health and style. Some can even help fix them for you , for more informations check this [https://www.freecodecamp.org/news/what-is-linting-and-how-can-it-save-you-time/](url)
- Kindly set up the github flow, it is a detailed process for using branches with your team and is important to collaborate more effectively because everyone knows how to track the history of changes in your shared code easily.
- to set up both linters and github flow please use this [https://github.com/microverseinc/linters-config](url)
- Kindly add a readme file ,The readme file is used to explain what is uploaded and how we can install or use it [https://github.com/microverseinc/readme-template](template) to set it by microverse standards.

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @AmineDerbal  👋 

## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉

 
 
 ## Highlights
 -  Descriptive Pull Request(PR) ✅ 
 -  Moderate review ✅ 
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
 - [ ] Though you did add two comments on the Summary, I strongly suggest you do make same on the inline comments also.
 
image

 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**